src/components/modules/FilterFacet.vue

```diff
@@ -252,13 +256,48 @@ export default {
       this.selectedBucketsIds = [];
       this.selectedBucketsItems = [];
     },
+    loadMoreBuckets() {
```

Comment:
It's a matter of style but it might be easier to  read if this function was `async` with `try..catch..finally` instead of promises. :)